javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,69 @@
+//Movies exercise
+
+const apiUri = `https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json`;
+
+async function movies() {
+  const response = await fetch(apiUri);
+  const dataMovies = await response.json();
+  console.log(dataMovies);
+
+  const badMovies = dataMovies.filter(function (params) {
+    return params.rating < 4;
+  });
+
+  console.log(badMovies);
+
+  const MoviesSinec = badMovies.filter(function (params) {
+    return params.year === 2000;
+  });
+
+  console.log(MoviesSinec);
+}
+movies();
+
+//Promise that resolves after set time
+
+const afterSeacond = new Promise((resolveAfter) => {
+  setTimeout(() => {
+    resolveAfter("I am the Promise");
+  }, 8000);
+});
+
+afterSeacond.then((elment) => {
+  console.log(elment);
+});
```

Comment:
`afterSeacond` is not a function, notice that you don't call it, you only add the `.then` handler. It is a Promise.

The exercise is to create a function, that takes an argument for the time to wait and that returns a Promise that resolves once it has waited the right amount of time.

So your interface should look like this:
`afterSeacond(8000).then((message) => console.log(message))`